src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+  
+Learn more about alerts [here](/docs/alerts-applied-intelligence/overview/).
+
+  </Side>
+  <Side>
+    <img
+      title="New Relic alerts"
+      alt="New Relic alerts"
+      src={NRAlertsMain}
+    />
+  </Side>
+  </SideBySide>
+
+# APM [#apm]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic APM"
+    alt="New Relic APM"
+    src={NRApmMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic instruments your application by using one of our many language agents. These agents collect metrics from your application. They send the metrics to <InlinePopover type="apm" />. It lets you monitor it using pre-built dashboards. 
+  
+  Learn more about APM [here](/docs/apm/new-relic-apm/getting-started/introduction-apm/).
+
+  </Side>
+  </SideBySide>
+
+# Browser [#browser]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="browser" /> capability collects data from the navigation timings in all end users' browsers. It sends that data directly to New Relic while blending with the service that served the requests. When you load that page URL, you can easily see which back-end transaction fulfilled that request. 
+
+  Learn more about browser [here](/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic browser"
+  alt="New Relic browser"
+  src={NRBrowserMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Dashboards [#dashboards]
+<SideBySide>
+  <Side>
+
+  <img
+  title="New Relic dashboards"
+  alt="New Relic dashboards"
+  src={NRDashboardsMain}
+/>
+
+  </Side>
+  <Side>
+
+  Our UI allows you to create custom charts and share them easily. Our <InlinePopover type="dashboards" /> capability lets you collect and visualize your data preferences across our platform. Tailor your charts to display the information you need. Access and analyze your data from any location within our platform.
+
+  NRQL, short for New Relic query language, is a powerful tool for accessing detailed data from New Relic. NRQL is like SQL. It lets you gather specific info about your applications, hosts, and business operations. By using NRQL, you can gain valuable insights into the performance and behavior of your environment. 
+
+  Learn more about dashboards [here](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) and about NRQL [here](/docs/nrql/get-started/introduction-nrql-new-relics-query-language/).
+  
+  </Side>
+  </SideBySide>
+
+# Errors inbox [#errors-inbox]
+<SideBySide>
+  <Side>
+
+  Errors inbox is an error-tracking solution. It's designed to put you in control. It helps you find, focus on, and fix errors across your application stack. Errors inbox is embedded in the <InlinePopover type="apm" /> UI so you can resolve errors faster.
```

Comment:
Errors inbox is an error-tracking solution. It's designed to put you in control.

To:

Errors inbox is an error-tracking solution designed to put you in control.